stressor/stress-test.mts

```diff
@@ -264,56 +309,96 @@ function findMode(arr: Array<Array<string>>): Array<string> {
  * Checks the results in a set of workflow runs for population and equality
  * @returns An object with percentages of populated and equal results
  */
-function checkRunSetResults(results: Array<WorkflowRunResults>) {
+function checkRunSetResults(runs: Array<WorkflowRun>): ComparisonRunResult {
   let totalRows = 0;
   let equalRows = 0;
 
-  const comparisonWorkflowRunResults = findMostCommonRunResults(results);
-
+  const comparisonWorkflowRunResults = findMostCommonRunResults(runs);
+  const comparedResults: Array<ComparisonTestRunResult> = [];
   comparisonWorkflowRunResults.forEach((compTest) => {
-    console.log(`\n### TestId ${compTest.testCsvRow}\n`);
-    console.log(`\n#### Comparison Set\n`);
-    console.log(`\`\`\`\n${compTest.screenreaderResponses}\n\`\`\``);
-    console.log(`\n#### Mismatches\n`);
-    results.forEach((result, i) => {
+    const { testCsvRow, screenreaderResponses: baselineResponses } = compTest;
+    // console.log(`\n### TestId ${testCsvRow}\n`);
+    // console.log(`\n#### Comparison Set\n`);
+    // console.log(`\`\`\`\n${baselineResponses}\n\`\`\``);
+    // console.log(`\n#### Mismatches\n`);
```

Comment:
```suggestion
```

leftover from first pass of markdown implementation, this section of code only processes now